From 3b4bf8366c6fe2e15798ab252cfeda76fb003105 Mon Sep 17 00:00:00 2001
From: Andrea Stacchiotti <andreastacchiotti@gmail.com>
Date: Tue, 9 Jan 2018 18:10:40 +0100
Subject: [PATCH] configure: Add `AM_PROG_CC_C_O` macro

This macro checks that the C compiler supports the -c and -o
options together, which is needed to build the testing exe.

This macro is needed only for automake <1.14, but add it for
compatibility with older environments.

Related: #289
Reported-by: @ChaosNicro

Last-Update: 2018-01-15

Patch-Name: fix-build-with-older-autotools.patch
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 47d1353..69bb587 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,6 +9,7 @@ AC_HEADER_STDBOOL
 LT_INIT
 
 IT_PROG_INTLTOOL
+AM_PROG_CC_C_O
 
 AC_CHECK_FUNCS(memset strcasecmp strchr strdup strerror strtoul getline)
 
